packages/app/features/dashboard/profile/likes/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { Text, View } from "app/design-system";
+import { useUserLikesQuery } from "app/api/graphql";
+import { isSome } from "app/utils";
+import { usePlayback } from "app/hooks/usePlayback";
+import { FlatList } from "react-native";
+import { BeatListEntry } from "app/ui/beat-list-entry";
+
+export default function LikesScreen() {
+  const { data } = useUserLikesQuery();
+  const entries = data?.userLikes?.filter(isSome) ?? [];
+  const { playEntry } = usePlayback();
+
```

Comment:
we need a state for loading and an empty list.

Reply:
Doing it in #33

Reply:
same as above: `can you do it here? #33 is about the collection feature, so we shouldn't include in it things connected with likes.`

Reply:
The code is already partly written there, this list is extracted there and used for both collection and likes. For the sake of easier merge we should leave it as is for now

Reply:
okay, but for the future remember to not mix code for different features in one pr.

Reply:
is it really mixing, here I created a simple list without loading and empty state, in the next pr i extracted it for use in the new feature and then refined it by adding loading and empty state